vars/env/dev/env.yml

```diff
@@ -485,3 +485,6 @@ env_django_logging: |
           }
       },
   }
+
+  # new relic
+  env_new_relic_license_key: "{{ vault_env_new_relic_license_key }}"
```

Comment:
I know this pattern is used everywhere, but is there any reason not to just, in all_var.yml, say:

new_relic_license_key: "{{ vault_env_new_relic_license_key }}"

Just asking in case this is an easy change. It would clean up these files a lot.

Reply:
Great idea. What I'm looking for is a way to avoid setting `new_relic_license_key`/installing the config for `DEV` and `TEST` (in order to minimize cost). Thinking about it though, I can use your suggestion and [change the conditional](https://github.com/CMSgov/bluebutton-web-deployment/pull/1254/files#diff-dc16b0cec89f5682e374bdfa8f15cd8fR8) to check `env`, only copying a `newrelic.ini` file over for `IMPL` or `PROD`.